app/models/section.rb

```diff
@@ -44,6 +44,10 @@ def name
     end
   end
 
+  def news_node
+    nodes.with_name('News')
```

Comment:
This will return a list of nodes rather than a single one, which is not what the name implies